code/modules/clothing/masks/gasmask.dm

```diff
@@ -11,6 +11,39 @@
 	flags_cover = MASKCOVERSEYES | MASKCOVERSMOUTH | PEPPERPROOF
 	resistance_flags = NONE
 
+/obj/item/clothing/mask/gas/alt
+	name = "gas mask"
+	desc = "A face-covering mask that can be connected to an air supply. Not good for concealing your identity or for blocking gas flow."
+	icon_state = "gas_alt2"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 10, "rad" = 0, "fire" = 0, "acid" = 0)
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	flags_inv = HIDEEARS|HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_NORMAL
+	item_state = "gas_alt2"
+
+/obj/item/clothing/mask/gas/alt2
+	name = "gas mask"
+	desc = "A face-covering mask that can be connected to an air supply. Not to be used for hiding eyes or blocking gas flow."
+	icon_state = "gas_alt3"
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	flags_inv = HIDEEARS|HIDEFACE|HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_NORMAL
+	gas_transfer_coefficient = 0.005 //not that this will ever be used but there you go
+	permeability_coefficient = 0.005
+	item_state = "gas_alt3"
+
+/obj/item/clothing/mask/gas/alt3
+	name = "gas mask"
+	desc = "A face-covering mask that cannot be connected to an air supply. Turns out to not be that great at blocking gasses."
+	icon_state = "gas_alt4"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 10, "rad" = 10, "fire" = 5, "acid" = 5)
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT

```

Comment:
i concur with the sentiments in the other thread honestly, having a gas mask that appears in an emergency fire closet that can't actually be used as internals is pretty shitty. If you want to conceal your identity, there's plenty of bandanas and crap laying around the station already that will do that for you.